src/pages/Vostfree/tests.json

```diff
@@ -0,0 +1,39 @@
+{
+  "title": "Vostfree",
+  "url": "https://vostfree.cx/",
+  "testCases": [
+    {
+      "url": "https://vostfree.cx/1188-summer-time-rendering-vostfr-ddl-streaming-1fichier-uptobox.html",
+      "expected": {
+        "sync": true,
+        "title": "Summer Time Rendering VOSTFR",
```

Comment:
you will have to remove stuff like VOSTFR and FRENCH from the title

Reply:
ok, it's done

Reply:
you didnt commit anything

Reply:
now it is

Reply:
seems like "VF" also has to be removed